.azure-pipelines/pull_request.yml

```diff
@@ -103,3 +103,4 @@ stages:
 
     jobs:
       - template: templates/documentation_build.yml
+      - template: templates/test_can_install_and_import.yml
```

Comment:
Not necessary for the PR?

Reply:
Why shouldn't it be tested for PRs?

Reply:
I though of this as a safeguard against publishing bad packages, but unlikely to fail, so I thought we could safe the time.

Reply:
The time is small and is less than the documentation build which runs parallel to them.

We'd actually need it in PR builds to guard against publishing bad packages, by the time this fails on the `master` job the bad package has already been published.